src/backends/kafka/mod.rs

```diff
@@ -201,24 +202,23 @@ impl<'a> ArroyoConsumer<'a, OwnedMessage> for KafkaConsumer {
     }
 
     fn commit_position(&mut self) -> Result<HashMap<Partition, Position>, ConsumerClosed> {
-        let mut map = HashMap::new();
+        let mut topic_map = HashMap::new();
         for (partition, position) in self.staged_offsets.iter() {
-            map.insert(
+            topic_map.insert(
                 (partition.topic.name.clone(), partition.index as i32),
                 Offset::from_raw(position.offset as i64),
             );
         }
 
-        match self.consumer.as_mut() {
-            None => {
-                return Err(ConsumerClosed);
-            }
-            Some(consumer) => {
-                let partitions = TopicPartitionList::from_topic_map(&map).unwrap();
-                let _ = consumer.commit(&partitions, CommitMode::Sync);
-            }
-        };
-        Ok(HashMap::new())
+        let consumer = self.consumer.as_mut().ok_or(ConsumerClosed)?;
+        let partitions = TopicPartitionList::from_topic_map(&topic_map).unwrap();
+        let _ = consumer.commit(&partitions, CommitMode::Sync).unwrap();
```

Comment:
When I tried to run the example last time commit did not happen at all, and I did not investigate, out of curiosity, does it work now ?

Reply:
Up to and including this line, there are no functional changes. Only below is actually difference. So I would say that is unlikely that commit behaviour changes. Though is the example actually supposed to commit? I don't see that happening.